packages/cli/src/commands/export-dynamic-plugin/backend-utils.ts

```diff
@@ -61,9 +67,15 @@ export function addToDependenciesForModule(
     );
     return;
   }
-  throw new Error(
-    `Several incompatible versions ('${existingDependencyVersion}', '${dependency.version}') of the same transitive dependency ('${dependency.name}') for embedded module ('${module}')`,
-  );
+  if (!ignoreVersionCheck.includes(dependency.name)) {
+    throw new Error(
+      `Several incompatible versions ('${existingDependencyVersion}', '${dependency.version}') of the same transitive dependency ('${dependency.name}') for embedded module ('${module}')`,
+    );
+  } else {
+    Task.log(
+      `Several incompatible versions ('${existingDependencyVersion}', '${dependency.version}') of the same transitive dependency ('${dependency.name}') for embedded module ('${module}') however this has been overridden`,
```

Comment:
Should we mention which version is finally kept in the peer dependencies, as it is done in the other messages above ?